src/bosh-dns/Gopkg.lock

```diff
@@ -7,6 +7,12 @@
   packages = [".","fakeclock"]
   revision = "2269160ae1757f96bbb8c6475e6fa36c805e73e0"
 
+[[projects]]
+  branch = "master"
+  name = "code.cloudfoundry.org/localip"
```

Comment:
can we somehow change the tests to not use localip?

Reply:
is the problem that we don't want to add another package? or something specific with how localip works?

Reply:
We'd have to copy https://github.com/cloudfoundry/localip/blob/master/localip.go#L6-L25 into the two tests in order to find the host's IP address as required for these tests. If you'd prefer we copy this in please let us know, @cppforlife.

Reply:
chatted with @dpb587-pivotal. how about using `spec.ip` in [1] and use env variable to pass that down to the test. this way we are not relying on additional libraries. localip has its uses but it's unfortunately does some interesting things with getting ip that for example does not work on ipv6 boxes. thoughts?

https://github.com/cloudfoundry/bosh-dns-release/blob/master/jobs/acceptance-tests-windows/templates/run.ps1

Reply:
I suppose we could do that for the acceptance tests since they're generally run via the errand. However, we also use localip in the unit tests here: https://github.com/cloudfoundry/bosh-dns-release/pull/14/files#diff-fa684476d19a968fda7db94e2bc58d82R147. I don't really like the idea of coupling tests to the environment if not absolutely necessary.

I hear the concerns about localip and IPv6, but for the purposes of test I think that's probably not a huge issue? I'd like to make it as easy as possible to run the tests and requiring additional setup seems unnecessary. We could just copy the localip logic into the two tests (or just units if you prefer we use `spec.ip` in acceptance), but at this point that seems like extra work for not much gain.